src/views/Package/PackageTypeDocs.tsx

```diff
@@ -0,0 +1,42 @@
+import { Heading } from "@chakra-ui/react";
+import { FunctionComponent } from "react";
+import { useParams } from "react-router-dom";
+import { Markdown } from "../../components/Markdown";
+import { PackageDocsError } from "./PackageDocsError";
+import { usePackageState } from "./PackageState";
+
+const usePackageTypeDocs = () => {
+  const { typeId }: { typeId?: string } = useParams();
+  const { apiReference } = usePackageState();
+
+  const content = apiReference?.[typeId ?? ""];
+  if (!typeId || !content) {
+    return;
+  }
+
+  return content;
```

Comment:
nit: Readability (I think this is equivalent, but could be wrong)
```suggestion
  if (typeId) {
    return apiReference?.[typeId];
  }

  return;
```